README.md

```diff
@@ -1,30 +1,35 @@
-## THE 2 COLLECTIONS
-This repository contains 2 Postman v2.1 collections shipped in JSON files that are ready to be imported into any local Postman instance. Collection ‘Mollie v2 API - API Key authentication’ is based on API Key authentication at Mollie and contains all endpoints that are available through that type of credentials. They are most applicable for a situation where there’s a one-to-one relation between a customer/merchant and Mollie, for instance one ecommerce platform instance that connects to one Mollie account. 
+# Overview
 
-The other collection, ‘Mollie v2 API - OAuth/access_token authentication’, is based on access_token credentials and includes the Mollie Connect feature set to retrieve those. Thus, this collection is tailored for SaaS / marketplace setups and customers/merchants that want to operate on account-level (Onboarding API, Administration API, etc.)
+This repository contains a set of Postman v2.1 collections to try out the functionality of the Mollie API. You can import the JSON files into any local Postman instance. The following collections are available:
```

Comment:
"collections to try out" --> "Collections that you can use to try out"

Reply:
```suggestion
This repository contains a set of Postman v2.1 collections that you can use to try out the functionality of the Mollie API. You can import the JSON files into any local Postman instance. The following collections are available:
```